### PR TITLE
rec-4.1.x: Add an option to export only responses over protobuf

### PR DESCRIPTION
**Pull request by rgacogne**
### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled this code
- [x] tested this code
- [x] included documentation (including possible behaviour changes)
- [ ] documented the code
- [x] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: pdns/recursordist/docs/lua-config/protobuf.rst [hunk 1]
```diff
@@ -12,7 +12,11 @@ Configuring Protocol Buffer logs
 --------------------------------
 Protobuf export to a server is enabled using the ``protobufServer()`` directive:
 
-.. function:: protobufServer(server [[[[[[[, timeout=2], maxQueuedEntries=100], reconnectWaitTime=1], maskV4=32], maskV6=128], asyncConnect=false], taggedOnly=false])
+.. function:: protobufServer(server [[[[[[[[, timeout=2], maxQueuedEntries=100], reconnectWaitTime=1], maskV4=32], maskV6=128], asyncConnect=false], taggedOnly=false], responsesOnly=false])
+
+  .. versionchanged:: 4.1.11
+
+    The optional ``responsesOnly`` parameter was added.
```
[the remark this answers is not in the document]
Comment (rgacogne) [reply pdns/recursordist/docs/lua-config/protobuf.rst]: Oh right, sorry! :+1:

[the remark this answers is not in the document]
Comment (aerique) [reply pdns/recursordist/docs/lua-config/protobuf.rst]: I'm not seeing the doc change in master.  Wanted to make a PR but the `protobufServer` definition is just `protobufServer(servers [, options]))` now.

[the remark this answers is not in the document]
Comment (Habbie) [reply pdns/recursordist/docs/lua-config/protobuf.rst]: The master docs (recursordist/docs/lua-config/protobuf.rst) have `[, options]` as 'new in 4.2.0' and `protobufServer(server[[[[[[[, timeout=2], maxQueuedEntries=100], reconnectWaitTime=1], maskV4=32], maskV6=128], asyncConnect=false], taggedOnly=false])` as 'deprecated in 4.2.0' - in other words, that's the 4.1.x version, and that's the one that got a new optional argument responsesOnly, at line 35 of protobuf.rst.

[the remark this answers is not in the document]
Comment (aerique) [reply pdns/recursordist/docs/lua-config/protobuf.rst]: Yes, but it's not clear to me if something still needs to be added to the docs in `master` as Pieter said above.

[the remark this answers is not in the document]
Comment (rgacogne) [reply pdns/recursordist/docs/lua-config/protobuf.rst]: The new option needs to be added to the 'deprecated' 4.1.x version of the docs in master, exactly at the place pointed out by Habbies, yes.

**Pull request merged**